### PR TITLE
Named exports

### PR DESCRIPTION
**Pull request by piuccio**
Same a #4 (kinda) and a fix for #9 

This code exports both a default and named exports

### CODE REVIEW

**Comment by developit** [comment]
This is awesome, will try to look at it and get it merged tonight.  If it works well I think we'll apply the same technique to Preact.

File: rollup.config.js [hunk 1]
```diff
@@ -18,13 +18,13 @@ const external = [
 ];
 
 export default {
-	exports: 'default',
```
Comment (developit) [inline rollup.config.js]: I think this results in a commonjs/umd bundle with exports like:

```js
exports.Provider = Provider;
exports.connect = connect;
exports.connectAdvanced = connectAdvanced;
exports.default = { Provider, connect, connectAdvanced };
```

Might be wrong though.

**Comment by anthonyjgrove** [comment]
updates?

**Comment by developit** [comment]
I need to merge just the src changes here, the build stuff will break commonjs exports.

**Comment by ezekielchentnik** [comment]
I think we're going to need two rollup configs.  

one for an es bundle (preact-redux.esm.js), the other for a pure UMD bundle (dist/preact-redux.js); this PR currently breaks commonjs exports - unless we are ok with forcing commonJs consumers to require this module using the default keyword e.g.

```javascript
const preactRedux = require('preact-redux').default
```

@developit not sure if you've concluded the same (maybe you found another solution).  If so i can tack onto this PR with what I'm suggesting

cheers

**Comment by ezekielchentnik** [comment]
see PR #15

**Comment by developit** [comment]
Fixed in 2.0.2.

**Pull request closed without merging**